worlds/alttp/__init__.py

```diff
@@ -544,6 +544,44 @@ def stage_extend_hint_information(cls, world, hint_data: typing.Dict[int, typing
                             er_hint_data[region.player][location.address] = main_entrance.name
         hint_data.update(er_hint_data)
 
+    @classmethod
+    def stage_modify_multidata(cls, multiworld, multidata: dict):
+
```

Comment:
this looks like it might break old seeds on new webhost and vice versa. what was tested? (i didn't test yet)

Reply:
the seeds themselves shouldn't break, the webtracker definitely does though. Current webtracker can't load rooms made with this change.